stats/stats_test.go

```diff
@@ -81,6 +83,52 @@ var (
 	}
 	// The id for which the service handler should return error.
 	errorID int32 = 32202
+    // Server Stats
```

Comment:
why do we need this extra comment `ServerStats`? Probably just remove since you have a sentence already for each. Applicable everywhere